mage_ai/server/server.py

```diff
@@ -557,7 +573,11 @@ def start_server(
     host: str = None,
     port: str = None,
     project: str = None,
+    manage: bool = False,
 ):
+    if manage:
+        os.environ[MANAGE_ENV_VAR] = '1'
+
```

Comment:
Let's not start scheduler for manage instance.

Reply:
Did you address this comment?

Reply:
oops i must have not saved the file